src/controllers/userController.js

```diff
@@ -64,6 +80,14 @@ const updateCurrentUser = async (req, res) => {
   attachCookiesToResponse({ res, user: tokenUser });
   // Send a JSON response with the status code 200 OK and the updated user
   res.status(StatusCodes.OK).json({ user });
+  /*
+     #swagger.summary = 'Fetch the requesting user, update their data and return the updated user.'
+     #swagger.responses[200] = {
+				description: 'User successfully fetched and updated.',
+				schema: { user: [{ $ref: '#/definitions/PasswordlessUser' }] }
```

Comment:
This should document returning a single user object not an array (under the `user` key)

```suggestion
				schema: { user: { $ref: '#/definitions/PasswordlessUser' } }
```